Themes/default/Help.template.php

```diff
@@ -29,7 +29,16 @@ function template_popup()
 	</head>
 	<body id="help_popup">
 		<div class="windowbg description">
-			', $context['help_text'], '<br>
+			';
+
+	if ($_GET['help'] == 'cal_short_months')
```

Comment:
It might be more appropriate to do this at the bottom of `Sources/Help.php`, but I wasn't sure which of the three cases I needed and I feared I'd have copy the `if` into each case. If I should move it there, tell me.

Reply:
yea It would definitely be much much better if you were to moves these conditional statements to the Source file.

The function you are looking for is `ShowAdminHelp()` located in `Help.php`. I haven't tested but I believe you want your code to be added before the last if statement in that function.

Instead of echoing you will just be assigning the result to `$context['help']`.

Also you might want to play with a `switch` block ?

Reply:
Indeed, a switch block would be preferable here.